### ISSUE TITLE
Doesn't support iPhone's higher then 8 plus

### ISSUE THREAD

**Issue by SarahH12099** [SarahH12099/MeSNEmu#1]
- Screen size is not full screen
- Controls are not properly placed

**Comment by lu1sprz** [comment]
Hey i have the fix for this one, can i push a branch with the fix so you can review it ? thanks !

**Comment by SarahH12099** [comment]
Yeah and thank you for helping with it, sorry about late response, never got a notification for this post

**Comment by ChoroGalaxiaJr** [comment]
![image0](https://user-images.githubusercontent.com/46771471/66789931-154ed780-eec4-11e9-9664-fb0f2507c081.png)
Has a release for the fix been made yet? on iPhone XR. Using latest release.

**Comment by SarahH12099** [comment]
> ![image0](https://user-images.githubusercontent.com/46771471/66789931-154ed780-eec4-11e9-9664-fb0f2507c081.png)
> Has a release for the fix been made yet? on iPhone XR. Using latest release. It's not really that big of a issue, but the screen is in the wrong location but the buttons seem fine.

No. I am working on a fix and it will hopefully be done for the 2.0.0.0 release, not guarenteed it will be fixed in time for the 2.0.0.0 release. The 2.0.0.0 release is set for hopefully 2 months from now.
